ynr/apps/people/models.py

```diff
@@ -33,6 +33,13 @@ def person_image_path(instance, filename):
     return "images/people/{0}/{1}".format(instance.person.id, filename)
 
 
+@unique
+class EditLimitationStatuses(Enum):
+    NO_STATUS = ""
+    LIABLE_TO_VANDALISM = "Liable to vandalism"
```

Comment:
Terminology: We used `NEEDS_REVIEW` in the LoggedActions PR. Do you think  that makes more sense here?

Reply:
So I was considering a future where we have pre-moderation as well as post-moderation, and `NEEDS_REVIEW` might be a term we use for that. 

As we don't have that, and actually have no plans to do so, I expect this is a good idea. We can always invent new terms if and when we decide on pre-moderation